src/services/WalletService/deterministic/hardware.ts

```diff
@@ -11,6 +11,7 @@ export interface ChainCodeResponse {
 export abstract class HardwareWallet extends DeterministicWallet implements IFullWallet {
   // Static functions can't be abstract, so implement an errorous one
   // @ts-ignore
```

Comment:
Can we remove the `@ts-ignore`?

Reply:
Removing `@ts-ignore` results in an unused variable error, and removing `dpath` results in errors elsewhere. The best option here would be to refactor it entirely, since there is no good reason to have `getChainCode` be static, but this is outside the scope of this PR.